libkmod/libkmod-module.c

```diff
@@ -586,12 +584,7 @@ KMOD_EXPORT const char *kmod_module_get_path(const struct kmod_module *mod)
 		return NULL;
 
 	/* lazy init */
-	line = kmod_search_moddep(mod->ctx, mod->name);
-	if (line == NULL)
-		return NULL;
-
-	kmod_module_parse_depline((struct kmod_module *)mod, line);
-	free(line);
+	module_get_dependencies_noref((struct kmod_module *)mod);
```

Comment:
No need to cast away the const here.

I see you're changing the signature with a latter commit. Let's introduce the cast with that commit.

Reply:
Adjusted